.github/workflows/lint.yaml

```
@@ -0,0 +1,21 @@
+name: Lint
+on:
+  push:
+jobs:
+  eslint:
+    name: Lint JavaScript files
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
```

Let's quote this number to prevent some [unintended side effects when using YAML numbers](https://ruudvanasseldonk.com/2023/01/11/the-yaml-document-from-hell).

```suggestion
          node-version: "18"
```